pkg/setting/setting.go

```diff
@@ -796,6 +797,7 @@ func (cfg *Cfg) Load(args *CommandLineArgs) error {
 		plug = strings.TrimSpace(plug)
 		cfg.PluginsAllowUnsigned = append(cfg.PluginsAllowUnsigned, plug)
 	}
+	cfg.MarketPlaceURL = pluginsSection.Key("marketplace_url").MustString("https://grafana.com/grafana/plugins/")
```

Comment:
nit picking: Marketplace is one word so p should not really be capitalized here

Reply:
Agree with @torkelo, didn't notice myself.